FeaturedArticleWidget/FeaturedArticleWidget.swift

```diff
@@ -2,13 +2,35 @@ import UIKit
 import NotificationCenter
 import WMF
 
+private final class EmptyView: SetupView {
+    private let label = UILabel()
+
+    var theme: Theme = .widget
+
+    override func setup() {
+        super.setup()
+        label.text = WMFLocalizedString("featured-article-empty-title", value: "No Featured article available today", comment: "Title that displays when featured article is not available")
```

Comment:
It looks like this isn't imported by the localization script... investigating

Reply:
thanks 🙌